### PR TITLE
Use full name for commands.Bot reference

### PR DESCRIPTION
**Pull request by nerdguyahmad**
## Summary

The docs seem to not show commands.Bot as hyperlink because it was mentioned as `commands.Bot` instead of `discord.ext.commands.Bot` This PR fixes it.

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Review by CodeWithSwastik — APPROVED** [review]

**Review by BobDotCom — APPROVED** [review]

**Comment by BobDotCom** [comment]
This will not be merged yet as it will create merge conflicts from our working branch. Once merge our working branch in, this will be implemented.

**Review by ToxicKidz — DISMISSED** [review]

**Review by Lulalaby — DISMISSED** [review]

File: discord/bot.py [hunk 1]
```diff
@@ -445,8 +445,7 @@ def command(self, **kwargs):
         """There is an alias for :meth:`application_command`.
 
         .. note::
-
```
Comment (Dorukyum) [inline discord/bot.py:448]: Why is this newline deleted?

Comment (nerdguyahmad) [reply discord/bot.py:448]: Newline isn't needed there

Comment (Dorukyum) [reply discord/bot.py:448]: There is a newline under every `.. note` in the source.

Comment (nerdguyahmad) [reply discord/bot.py:448]: okithen

File: discord/bot.py [hunk 2]
```diff
@@ -445,8 +445,8 @@ def command(self, **kwargs):
...
-            This decorator is overridden by :class:`commands.Bot`.
+
```
Comment (Dorukyum) [inline discord/bot.py:448]: It has extra whitespace now 😅
```suggestion

```

File: discord/bot.py [hunk 3]
```diff
@@ -445,8 +445,7 @@ def command(self, **kwargs):
         """There is an alias for :meth:`application_command`.
 
         .. note::
-
-            This decorator is overridden by :class:`commands.Bot`.
+            This decorator is overridden by :class:`discord.ext.commands.Bot`.
```
Comment (Dorukyum) [inline discord/bot.py:448]:
```suggestion

            This decorator is overridden by :class:`discord.ext.commands.Bot`.
```

**Comment by Dorukyum** [comment]
Somehow couldn't add the newline back but it can be done later. This small pr is getting too long.

**Review by Dorukyum — APPROVED** [review]

**Comment by nerdguyahmad** [comment]
> This small pr is getting too long.

I'm wondering that too. What's wrong :weary:

**Review by Lulalaby — APPROVED** [review]

**Pull request merged**